python/tree_plotting.py

```diff
@@ -831,85 +830,92 @@ def draw_plot(plotDict, plotname, lumi, year, logY=True, logX=False, plotData=Fa
     # Diphoton mass cut
     #cut = "Diphoton_mass > 95"
 
+    # Photon pT / Mgg cuts
+    #cut = "LeadPhoton_pt/Diphoton_mass > 0.33"
+    #cut = "SubleadPhoton_pt/Diphoton_mass > 0.25"
+    #cut = "LeadPhoton_pt/Diphoton_mass > 0.33 && SubleadPhoton_pt/Diphoton_mass > 0.25"
+
     cut = weight + "*(" + cut + ")"
 
 
     # List of plots
-    plotNames = []; plotBins = dict(); plotXTitles = dict(); plotYTitles = dict()
+    plotNames = dict(); plotBins = dict(); plotXTitles = dict(); plotYTitles = dict()
```

Comment:
What is the difference of changing from a list into a dict in this line and the following?

Reply:
As mentioned in the description of the PR, this
 > - Allows a different name for a plot than the expression that is used to construct it.

The key of the dictionary is the unique plot identifier, while the contained item is the expression used to construct it. If the contain item string is empty, then the dictionary key is used as the expression to construct the plot. You can read the code to understand how this works. The best example to understand this would be these two lines:
https://github.com/cmstas/XToYggHbb_looper/blob/ea9cb2f04c3c34f13b28b7b79ed403da819a72a0/python/tree_plotting.py#L875-L876

Reply:
Thank you Manos! I understand.